src/experimental/EmailAccountRecoveryNew.sol

```diff
@@ -12,9 +12,15 @@ import { ERC1967Proxy } from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy
 /// new guardian and recovering a wallet.
 abstract contract EmailAccountRecoveryNew {
     uint8 constant EMAIL_ACCOUNT_RECOVERY_VERSION_ID = 1;
-    address public verifierAddr;
-    address public dkimAddr;
-    address public emailAuthImplementationAddr;
+    address internal immutable verifierAddr;
```

Comment:
these have getter functions. so will reduce deployment bytecode size

Reply:
makes sense, posted this suggestion in https://github.com/zkemail/ether-email-auth/pull/28